src/plugins/data_views/public/data_views/data_views_api_client.test.ts

```diff
@@ -23,8 +23,26 @@ describe('IndexPatternsApiClient', () => {
   });
 
   test('uses the right URI to fetch fields for wildcard', async function () {
-    await indexPatternsApiClient.getFieldsForWildcard({ pattern: 'blah' });
+    await indexPatternsApiClient.getFieldsForWildcard({ pattern: 'blah', allowHidden: false });
 
-    expect(fetchSpy).toHaveBeenCalledWith(expectedPath, expect.any(Object));
+    expect(fetchSpy).toHaveBeenCalledWith(expectedPath, {
+      // not sure what asResponse is but the rest of the results are useful
```

Comment:
I think `asResponse` just prevents the HTTP service from automatically parsing the response as JSON so we can deal with it in its raw form (which makes sense since we're doing ETags, etc.).